src/twosection.jl

```diff
@@ -0,0 +1,85 @@
+"""
+    TwoSectionView{T<:Real} <: AbstractGraph{Int64}
+
+Create a 2-section view of a hypergraph `h`.
+Note this is a view - changes to the original hypergraph will be automatically reflected in the view.
+
+The 2-section view of a hypergraph is suitable for processing with the LightGraphs.jl package.
+Several LightGraphs methods are provided for the compability.
+
+"""
+struct TwoSectionView{T<:Real} <: AbstractGraph{Int}
+    h::Hypergraph{T}
+    function TwoSectionView{T}(h::Hypergraph{T}) where {T<:Real}
+        new(h)
+    end
+end
+
+LightGraphs.nv(t::TwoSectionView) = length(t.h.v2he)
+
+LightGraphs.vertices(t::TwoSectionView) = Base.OneTo(nv(t))
+
+LightGraphs.ne(t::TwoSectionView)::Int64 = 0.5*sum(length.(t.h.he2v) .* (length.(t.h.he2v) .- 1))
```

Comment:
@pszufe: can you please benchmark which implementation is faster. The one in the code or the one below?
```suggestion
function LightGraphs.ne(t::TwoSectionView)
    s = 0
    for x in t.h.he2v
        s += length(x) * (length(x) - 1)
    end
    div(s, 2)
end
```